### PR TITLE
[linkedin conversions] Adds lookback window fields to hook inputs + adds new conversion type options

### PR DESCRIPTION
**Pull request by nick-Ag**
This PR updates the LinkedIn Conversions API destination with new hook input fields that are required to create a conversion rule. It also adds all possible options for the existing conversion type field so that users can select them from the dropdown.

## Testing
Tested successfully in local + staging.

Conversion Rule created:
Screenshot 2024-03-11 at 12 14 47 PM

Conversion Rule in LinkedIn manager:
Screenshot 2024-03-11 at 2 37 24 PM

### Staging
Conversion Type fields correctly displayed:
Screenshot 2024-03-11 at 3 15 27 PM

The newly added fields:
Screenshot 2024-03-11 at 3 32 58 PM

The conversion rule is successfully created with the lookback window params:
Screenshot 2024-03-11 at 3 35 45 PM

That same rule in LinkedIn:
Screenshot 2024-03-11 at 3 36 00 PM

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/linkedin-conversions/constants.ts [hunk 1]
```diff
@@ -8,3 +8,54 @@ export const SUPPORTED_ID_TYPE = [
   'ACXIOM_ID',
   'ORACLE_MOAT_ID'
 ]
+
+interface Choice {
```
Comment (marinhero) [inline packages/destination-actions/src/destinations/linkedin-conversions/constants.ts:12]: 👍🏽

File: packages/destination-actions/src/destinations/linkedin-conversions/api/index.ts [hunk 2]
```diff
@@ -53,7 +55,9 @@ export class LinkedInConversions {
           id: conversionRuleId,
           name: data.name || `No name returned for rule: ${conversionRuleId}`,
           conversionType: data.type || `No type returned for rule: ${conversionRuleId}`,
-          attribution_type: data.attributionType || `No attribution type returned for rule: ${conversionRuleId}`
+          attribution_type: data.attributionType || `No attribution type returned for rule: ${conversionRuleId}`,
+          post_click_attribution_window_size: data.postClickAttributionWindowSize || 30,
```
Comment (marinhero) [inline packages/destination-actions/src/destinations/linkedin-conversions/api/index.ts:59]: Consider moving the numbers to constants that denote what they mean.

**Review by marinhero — APPROVED** [review]

File: packages/destination-actions/src/destinations/linkedin-conversions/api/index.ts (hunk 2 above)
Comment (nick-Ag) [reply packages/destination-actions/src/destinations/linkedin-conversions/api/index.ts:59]: good call 👍

**Comment by nick-Ag** [comment]
@joe-ayoub-segment , no need to push this one, I still need to merge in https://github.com/segmentio/action-cli/pull/157 so will handle it when that is complete

**Comment by joe-ayoub-segment** [comment]
Hi @nick-Ag PR deployed - no Push

**Pull request merged**